config/locales/en.yml

```diff
@@ -121,3 +121,28 @@ en:
         roles_and_permissions:        'Users and groups'
         system_status:                'System status'
         technical:                    'Technical'
+  proprietor:
```

Comment:
No, you're right, @tdonohue, and I now realize that we do *not* sort our YAML files in Hyku. Maybe something for a later date. Thanks!